### PR TITLE
vso #5713642 multi ipconfig for nic

### PR DESCRIPTION
**Pull request by skomarag**
HI,
This primarily included adding a new field to dataconteracts called "primary" and adding tests and modifying existing tests primarily for network and compute solutions

with tests and code contract changes
in the network solution
Added more tests in compute.
All tests are passing in both record and play back mode.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@skomarag__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (skomar). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by markcowl** [comment]
@azuresdkci add to whitelist

**Comment by markcowl** [comment]
@deepakrajendranmsft @huangpf @hyonholee These changes lookm OK to me, can you take a look and give a quick signoff?

File: src/ResourceManagement/Network/Microsoft.Azure.Management.Network/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -19,7 +19,7 @@
 [assembly: AssemblyTitle("Microsoft Azure Network Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Network management functions for managing the Microsoft Azure Network service.")]
 
-[assembly: AssemblyVersion("4.0.0.0")]
+[assembly: AssemblyVersion("4.6.0.0")]
 [assembly: AssemblyFileVersion("4.5.2.0")]
```
Comment (hovsepm) [inline src/ResourceManagement/Network/Microsoft.Azure.Management.Network/Properties/AssemblyInfo.cs]: you will need AssemblyFileVersion to be higher or equal to AssemblyVersion

**Comment by DeepakRajendranMsft** [comment]
@markcowl We did an internal review of this already LGTM

**Comment by skomarag** [comment]
hi @hovsepm  : pushed the change with new assembly file version... please merge it after u review

**Comment by skomarag** [comment]
HI please see my PR and merge the changes

**Comment by skomarag** [comment]
HI Does this look good.. let me know otherwise

**Comment by huangpf** [comment]
Please wait for @hyonholee's signoff. Thanks.

**Comment by hovsepm** [comment]
> using System;

please add license header to this file

---
Refers to: src/ResourceManagement/Network/Network.Tests/Helpers/FeaturesInfo.cs:1 in 265cee8. [](commit_id = 265cee87055b464cdbc62f8098b0c1353f31bbf1, deletion_comment = False)

File: src/ResourceManagement/Network/Network.Tests/Tests/NetworkInterfaceTests.cs [hunk 2]
```diff
@@ -1,5 +1,8 @@
 ﻿using System.Collections.Generic;
 using System.Net;
+using System;
```
Comment (hovsepm) [inline src/ResourceManagement/Network/Network.Tests/Tests/NetworkInterfaceTests.cs]: Please add license header to this file as well.

**Pull request merged**